### PR TITLE
added average hh income for people of color

### PR DESCRIPTION
**Pull request by maggiesu0725**

### CODE REVIEW

**Comment by maggiesu0725** [comment]
@lhendey  ready for review

File: Prog/Equity_Compile_Feature_Data.sas [hunk 1]
```diff
@@ -62,7 +70,7 @@ data create_flags;
 	end;
 
 	else do;
-			 if PITI_First <= (72935/ 12*.28) then AMI_first_afford=1; else AMI_first_afford=0; *DC Median income 2016 ACS;
+			 if PITI_First <= (68362/ 12*.28) then AMI_first_afford=1; else AMI_first_afford=0; *DC Median income 2016 ACS for people of color 68362;
```
Comment (lhendey) [inline Prog/Equity_Compile_Feature_Data.sas]: update note to average household

Comment (maggiesu0725) [reply Prog/Equity_Compile_Feature_Data.sas]: just updated it

**Comment by lhendey** [comment]
@maggiesu0725 can you also change the label for the variable that gets output?

**Comment by maggiesu0725** [comment]
sorry about that, all labels for sale affordability should be updated @lhendey

**Pull request merged**